dependencies.yaml

```diff
@@ -111,6 +125,8 @@ dependencies:
           # pre-commit requires identify minimum version 1.0, but clang-format requires textproto support and that was
           # added in 2.5.20, so we need to call out the minimum version needed for our plugins
           - identify>=2.5.20
+      - output_types: conda
```

Comment:
#1344 is a duplicate, we can close that when this merges (or merge that before this PR, either way)